client/src/components/MarkdownDisplay/MarkdownDisplay.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react';
+import ReactMarkdown from 'react-markdown';
+
+type MarkdownDisplayProps = {
+  markdown: string;
+};
+
+const MardownDisplay: FC<MarkdownDisplayProps> = ({ markdown }) => {
+  const cleanedMarkdown = markdown
```

Comment:
I initially thought managing this in the ui was the way to go, but Rob managed to convince me that it belongs in the backend. That way we will be returning good markdown via the api. We also need to preserve (relative) indentation. You should be able to [copy this code](https://github.com/onc-healthit/inferno-program/blob/9bc10ff063ac70ce7b32475ed2d4200f3eaa637b/lib/app/helpers/browser_logic.rb#L62) from current inferno to handle the indentation.